Source/WebCore/platform/audio/cocoa/MediaSessionManagerCocoa.mm

```diff
@@ -59,7 +61,7 @@
 #endif // !PLATFORM(MAC)
 
 MediaSessionManagerCocoa::MediaSessionManagerCocoa()
-    : m_nowPlayingManager(platformStrategies()->mediaStrategy().createNowPlayingManager())
+    : m_nowPlayingManager(hasPlatformStrategies() ? platformStrategies()->mediaStrategy().createNowPlayingManager() : nullptr)
```

Comment:
that doesn't seem to be related to this change

Reply:
This will crash without this change as `platformStrategies()` returns NULL in the GPU process.